### PR TITLE
Feature update study

### PR DESCRIPTION
**Pull request by pelefebvre**
Adding a new feature on lpnamer : 
* Update the antares study with the antares xpansion solution

### CODE REVIEW

File: src/src_cpp/src_helpers/ortools_utils.cc [hunk 1]
```diff
@@ -6,6 +6,7 @@
 #include "ortools/linear_solver/linear_solver.pb.h"
 #include "ortools/linear_solver/model_exporter.h"
 
+//TODO rename to .cpp
```
Comment (flomnes) [inline src/src_cpp/src_helpers/ortools_utils.cc:9]: execute TODO

File: src/src_cpp/src_lpnamer/LinkdataRecord.cpp [hunk 2]
```diff
@@ -0,0 +1,98 @@
+#include "StudyUpdater.h"
+
+
+
+LinkdataRecord::LinkdataRecord(bool modernAntaresVersion_p):
```
Comment (flomnes) [inline src/src_cpp/src_lpnamer/LinkdataRecord.cpp:5]: What is a "modern" version of Antares ? Do we really want to use an older version of Antares ?

File: src/src_cpp/src_lpnamer/StudyUpdater.cpp [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    //default
+    std::cout << "StudyUpdater::readAntaresVersion : default version 710 returned.\n";
+    antaresVersion_ = 710;
+}
+
+
+
+std::string StudyUpdater::getLinkdataFilepath(Candidate const & candidate_p) const
+{
+	std::string result_l = linksPath_ + PATH_SEPARATOR + candidate_p.str("linkor") + PATH_SEPARATOR + candidate_p.str("linkex") + ".txt";
```
Comment (flomnes) [inline src/src_cpp/src_lpnamer/StudyUpdater.cpp:53]: consider using `std::filesystem` ?

File: src/src_python/antares_xpansion/driver.py [hunk 4]
```diff
@@ -218,7 +220,12 @@ def launch(self):
         if self.config.step == "full":
             lp_path = self.generate_mps_files()
             self.launch_optimization(lp_path)
+<<<<<<< HEAD
```
Comment (flomnes) [inline src/src_python/antares_xpansion/driver.py:223]: This probably should not be there

**Review by flomnes — CHANGES_REQUESTED** [review]
Please solve conflict in driver.py, at lines 223 and 225-228

File: src/src_cpp/src_test/CMakeLists.txt [hunk 5]
```diff
@@ -37,14 +40,9 @@ target_link_libraries (unit_tests PRIVATE
...
 		lpnamer
+		${PROJECT_NAME}::sequentialcore
 	)
 
-if (USE_SEQUENTIAL)
-	target_link_libraries (unit_tests PRIVATE
-			${PROJECT_NAME}::sequentialcore
-	)
-endif ()
-
```
Comment (jmkerloch) [inline src/src_cpp/src_test/CMakeLists.txt:47]: `USE_SEQUENTIAL` could still be defined to `OFF `or `USE_SEQUENTIAL` is not needed anymore.
Need to keep link to `sequentialcore` only if used and needed.

File: src/src_python/antares_xpansion/driver.py [hunk 6]
```diff
@@ -253,9 +263,18 @@ def clear_old_log(self):
...
             and (os.path.isfile(self.exe_path(self.config.LP_NAMER) + '.log')):
             os.remove(self.exe_path(self.config.LP_NAMER) + '.log')
+=======
+        if (self.args.step in ["full", "lp"])\
+            and (os.path.isfile(self.exe_path(self.config.LP_NAMER) + '_generate.log')):
+            os.remove(self.exe_path(self.config.LP_NAMER) + '_generate.log')
+        if (self.args.step in ["full", "update"])\
+            and (os.path.isfile(self.exe_path(self.config.LP_NAMER) + '_update.log')):
+            os.remove(self.exe_path(self.config.LP_NAMER) + '_update.log')
+>>>>>>> origin/master
```
Comment (jmkerloch) [inline src/src_python/antares_xpansion/driver.py:277]: Need correction of merge

**Review by jmkerloch — CHANGES_REQUESTED** [review]

**Pull request merged**